pages/index.tsx

```diff
@@ -18,7 +18,7 @@ import MenuIcon from "@material-ui/icons/Menu";
 import Header from "../features/core/Header";
 import ContractState from "../features/contract-state/ContractState";
 import ManagePosition from "../features/manage-position/ManagePosition";
-import EmpSelector from "../features/emp-selector/EmpSelector";
```

Comment:
Is this file deprecated now?

Reply:
Yea its deprecated. Should i just delete it?

Reply:
Yeah why not